server/db/migrations/20250315123278-add-role-id-to-users.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
```

Comment:
there is no migration to move the existing admin roles to the new one. For example what happens to a user who is currently a supervisor with is_supervisor true field?